cypress/integration/test_customer.js

```diff
@@ -1,13 +1,37 @@
 
-context('Customer', () => {
+
+context('Create Customer', () => {
 	before(() => {
 		cy.login();
 	});
-	it('Check Customer Group', () => {
-		cy.visit(`app/customer/`);
-		cy.get('.primary-action').click();
-		cy.wait(500);
-		cy.get('.custom-actions > .btn').click();
+
+	it('Create Customer', () => {
+		cy.visit(`app/customer`);
+		cy.wait(200);
+		cy.click_listview_primary_button('Add Customer');
+		cy.contains('Edit in full page').click();
+		cy.location("pathname").should("eq","/app/customer/new-customer-1");
+		cy.get_field('customer_name', 'Data').type('Nidhi', {delay: 200});
+		cy.get_field('customer_type', 'Select').should('have.value', 'Company');
+		cy.get_field('customer_group', 'Link').clear().type('All Customer Groups');
+		cy.get_field('territory', 'Link').clear().type('All Territories');
+		cy.get('.form-page > :nth-child(5) > .section-head > .ml-2 > .icon > .mb-1').click(); //click to expand 'Currency and Price List' section
```

Comment:
Try to target the section by the text "currency and price list" instead of the selector. 

The selector will eventually change and this test will start failing.